chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileTps.java

```diff
@@ -0,0 +1,116 @@
+package com.griddynamics.jagger.user.test.configurations.load;
+
+import com.griddynamics.jagger.user.test.configurations.load.auxiliary.TransactionsPerSecond;
+
+import java.util.Objects;
+
+/**
+ * This type of load implements an exact number of transactions per second performed by Jagger.
+ * Available attributes:
+ *     - transactionsPerSecond - A goal number of transactions per second
+ *
+ * Optional attributes:
+ *     - maxLoadThreads - Maximum number of parallel threads allowed for load generation
+ *     - warmUpTimeInMilliseconds - Load will increase from 0 to @e transactionsPerSecond in this time
+ *
+ * @ingroup Main_Load_profiles_group
+ */
```

Comment:
please add @@details section to documentation with the description of the difference between TPS and RPS (take from the old docu). add it to RPS docu as well